test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -855,9 +855,9 @@ def revert_server(self, server, force=False):
         server_stat = server.status(SERVER)[0]
         current_user = pwd.getpwuid(os.getuid())[0]
         try:
-            # remove current user's entry from managers list (if any)
-            a = {ATTR_managers: (DECR, current_user + '@*')}
-            server.manager(MGR_CMD_SET, SERVER, a, sudo=True)
+            # Unset managers list
+            a = ['managers']
+            server.manager(MGR_CMD_UNSET, SERVER, a, sudo=True)
```

Comment:
Can we add expect=True here as well as below server.manager()?

Reply:
I suspect it has to do with the fact that you're incrementing two things at once.  I suspect if you broke it out into two manager calls, expect=True would work.  I'd call this a PTL issue.  Of course I'm not exactly sure what PTL should do with it.  You're adding two elements to a string array.  I guess PTL could break it on the comma and check if both are in the string.

Reply:
Yes this is expect issue... @borlesanket Can you please file an issue for this?

Reply:
I have logged PP-1277 for this issue.